src/recipe.js

```diff
@@ -22,18 +22,13 @@ class Recipe {
     return totalCost;
   }
 
-  filterAllRecipes(recipe, tag) {
-    if (recipe.tags.includes(tag)) {
-      return recipe
+  determineRecipeTag(tag) {
+    if (this.tags.includes(tag)) {
+      return true
+    } else {
+      return false
```

Comment:
Great catch with incorporating the Recipe class into our code. I think renaming the function also allows us to know exactly what its purpose is.

Reply:
I love the rename here. It much more empathic.